components/app/myrw/datasets/pages/my-rw-datasets/dataset-list/dataset-list-component.js

```diff
@@ -47,7 +45,7 @@ class DatasetsList extends PureComponent {
 
     toastr.confirm(
       `Are you sure you want to delete the dataset: ${
-      metadata && metadata.attributes.info ? metadata.attributes.info.name : dataset.name
+        metadata && metadata.attributes.info ? metadata.attributes.info.name : dataset.name
```

Comment:
We would need to update the fetching of this component... It seems is still using the deprecated service as we have `attributes` here 🙄.  I'll add a task on PT: https://www.pivotaltracker.com/story/show/166727155